python/semantic_kernel/connectors/ai/open_ai/prompt_execution_settings/azure_chat_prompt_execution_settings.py

```diff
@@ -159,6 +159,7 @@ class ExtraBody(KernelBaseModel):
     data_sources: list[DataSource] | None = None
     input_language: Annotated[str | None, Field(serialization_alias="inputLanguage")] = None
     output_language: Annotated[str | None, Field(serialization_alias="outputLanguage")] = None
+    user_security_context: Annotated[str | None, Field()] = None
```

Comment:
If we don't need any special serialization alias handling for this attribute, we can remove the need for the empty `Field()`.

```suggestion
    user_security_context: str | None = None
```